openownership/data-standard
# Simon 369 interest types

# Overview

- What does this pull request do?
Implements the feature ticket #369 

- How can a reviewer test or examine your changes?
Review the issues marked COMMIT in the description of the feature ticket and check that the codelist has been updated as described in those issues. Check the changelog commit.

- Who is best placed to review it?
@kd-ods 

(Closes/Relates to) issue: #

## Translations

      translating. See: https://openownership.github.io/bods-dev-handbook/translations.html

## Documentation & Release

      https://openownership.github.io/bods-dev-handbook/standard_releases.html
